libs/shared/components/src/lib/button/button.tsx

```diff
@@ -15,7 +17,11 @@ export function Button(props: ButtonProps) {
         color: props.disabled ? 'gray' : props.color,
         borderColor: props.disabled ? 'gray' : props.color,
         opacity: props.disabled ? '0.5' : '1',
-      }}
+        marginLeft: props.center ? 'auto' : 'unset',
```

Comment:
As I said, i like the idea, but dont cherrypick the css props unless it's really needed,